legend-depot-artifacts-purge/src/main/java/org/finos/legend/depot/store/artifacts/purge/ArtifactsPurgeModule.java

```diff
@@ -30,4 +39,13 @@ protected void configure()
 
         bind(ArtifactsPurgeService.class).to(ArtifactsPurgeServiceImpl.class);
     }
+
+    @Provides
+    @Singleton
+    @Named("delete-versions-not-in-repository")
+    boolean initDeleteVersionsNotInRepository(SchedulesFactory schedulesFactory, ArtifactsPurgeService purgeService, ArtifactRepositoryProviderConfiguration configuration)
+    {
+        schedulesFactory.register(DELETE_VERSIONS_NOT_IN_REPO, LocalDateTime.now().plusSeconds(configuration.getVersionsUpdateIntervalInMillis() / 1000), configuration.getFixVersionsMismatchIntervalInMillis(), false,() -> purgeService.deleteVersionsNotInRepository());
```

Comment:
we should have a separate config for this

Reply:
added separate config attribute